### ISSUE TITLE
[FEATURE] Save time stamp of downloaded videos

### ISSUE THREAD

**Issue by ThaumicMekanism** [eni9889/YT-PP-Issues#107]
I forget if there is another post already about this but is it possible for you to make it so videos keep their time if you quit it mid-video. This could save the data both locally and also on youtube side since youtube already save timestamps. Maybe if you do both an offline (just on the device) and online (through how youtube saves your current time) storage of the timestamp that would integrate nicely with the other request I had about shown 'ghost' videos on a different device than what the videos were downloaded on for easy device swapping. 

tl;dr Make it so videos saved time spots and synced it with youtube so it is saved both offline and online. Also make it so it updates if the online is further ahead of the offline.

Thanks again for your awesome support. Please feel free to contact if you have any questions or concerns with this request. :)
